webui/react/src/e2e/tests/projectsWorkspaces.spec.ts

```diff
@@ -19,129 +14,199 @@ test.describe('Projects', () => {
+          'fromButton',
+        );
+
+        await sidebar.workspaces.pwLocator.click();
+        await sidebar.sidebarWorkspaceItem(wsCreatedWithButton).pwLocator.waitFor();
+        await workspaceListSetup.cardByName(wsCreatedWithButton).pwLocator.waitFor();
+      });
+      await authFixtureSetup.logout();
+      await pageSetup.close();
+    });
 
-  test('Projects and Workspaces CRUD', async ({ authedPage }) => {
-    const workspaceList = new WorkspaceList(authedPage);
-    const workspaceDetails = new WorkspaceDetails(authedPage);
+    test.afterAll(async ({ browser, dev }) => {
+      const pageTeardown = await browser.newPage();
+      await dev.setServerAddress(pageTeardown);
+
+      const authFixtureSetup = new AuthFixture(pageTeardown);
+      await authFixtureSetup.login();
+      const workspaceListTeardown = new WorkspaceList(pageTeardown);
+      await workspaceListTeardown.goto();
+      const deleteModal = workspaceListTeardown.deleteModal;
+
+      await test.step('Delete Workspace from Card', async () => {
+        if (wsCreatedWithButton !== '') {
+          await workspaceListTeardown.nav.sidebar.workspaces.pwLocator.click();
+          const workspaceCard = workspaceListTeardown.cardByName(wsCreatedWithButton);
+          await (await workspaceCard.actionMenu.open()).delete.pwLocator.click();
+          await deleteModal.nameConfirmation.pwLocator.fill(wsCreatedWithButton);
+          await deleteModal.footer.submit.pwLocator.click();
+        }
+      });
+      await authFixtureSetup.logout();
+      await pageTeardown.close();
+    });
 
-    await test.step('Create a Workspace from Card', async () => {
-      await workspaceList.newWorkspaceButton.pwLocator.click();
-      wsCreatedWithButton = await createWorkspaceAllFields(workspaceList.createModal, 'fromButton');
+    test('Attempt to delete a workspace but with bad validation', async ({ authedPage }) => {
+      const workspaceList = new WorkspaceList(authedPage);
+      const deleteModal = workspaceList.deleteModal;
 
-      await expect(
-        workspaceList.nav.sidebar.sidebarWorkspaceItem(wsCreatedWithButton).pwLocator,
-      ).toBeVisible();
       await workspaceList.nav.sidebar.workspaces.pwLocator.click();
-      await workspaceList.cardByName(wsCreatedWithButton).pwLocator.waitFor();
+      const workspaceCard = workspaceList.cardByName(wsCreatedWithButton);
+      await (await workspaceCard.actionMenu.open()).delete.pwLocator.click();
+      await deleteModal.nameConfirmation.pwLocator.fill('bad validation');
+      await expect(deleteModal.footer.submit.pwLocator).toBeDisabled();
+    });
+
+    test.describe('Project', () => {
+      let projectOneName = '';
+
+      test.beforeAll(async ({ browser, dev }) => {
+        const pageSetup = await browser.newPage();
+        await dev.setServerAddress(pageSetup);
+        const authFixtureSetup = new AuthFixture(pageSetup);
+        await authFixtureSetup.login();
```

Comment:
All other things being equal, it'll be faster to log in with the api. Doesn't matter much for one test, but think we should be defaulting to logging in with the api given the choice.